app/controllers/notes_controller.rb

```diff
@@ -96,6 +96,10 @@ def show
       @node = DrupalNode.find params[:id]
     end
 
+    if @node.has_power_tag('question')
+      redirect_to @node.path(:question)
+    end
+
```

Comment:
This redirects any question node to the question url to avoid two versions of a question.